src/components/About.jsx

```diff
@@ -0,0 +1,15 @@
+const React = require('react');
+
+const About = React.createClass({
+    render: function() {
+      return (
+        <div>
+          <div>
+            <h1>About</h1>
+          </div>
+        </div>
+      );
+  }
+});
+
```

Comment:
Straightforward `About` component. May benefit by having a navbar, otherwise I wouldn't have known it existed.